### PR TITLE
Enable ImageBitmap if interface is present

### PR DESCRIPTION
**Pull request by slimbuck**
Since we no longer flip images at load time in the engine (#3335) we can use ImageBitmap for loading images.

ImageBitmap performs more work off the main thread and should result in much smoother experiences for our users.

This PR was tested on a recent version of Chrome, Safari and Firefox.

### CODE REVIEW

**Comment by mvaligursky** [comment]
any concerns on older browses? For example:
- Firefox doesn't support options parameter to createImageBitmap (see https://bugzilla.mozilla.org/show_bug.cgi?id=1533680)

**Comment by slimbuck** [comment]
> any concerns on older browses? For example:
> 
> * Firefox doesn't support options parameter to createImageBitmap (see https://bugzilla.mozilla.org/show_bug.cgi?id=1533680)

Nope. That was an issue with 'flipY' option which we no longer need.

**Comment by slimbuck** [comment]
Think I will try find an old version of firefox to test on though.

File: package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
   ],
   "license": "MIT",
   "main": "build/playcanvas.js",
-  "module": "build/playcanvas.mjs",
+  "module": "build/playcanvas.mjs/index.js",
```
Comment (mvaligursky) [inline package.json:21]: keep this as a separate PR?

File: src/deprecated/deprecated.js [hunk 2]
```diff
@@ -904,7 +904,7 @@ Object.defineProperty(XrInputSource.prototype, 'rotation', {
 
 // INPUT
 
-export const input = {
+export const input_aa = {
```
Comment (mvaligursky) [inline src/deprecated/deprecated.js:907]: you sure?

File: package.json (hunk 1 above)
Comment (slimbuck) [reply package.json]: oops, unintended: reverted.

File: src/deprecated/deprecated.js (hunk 2 above)
Comment (slimbuck) [reply src/deprecated/deprecated.js]: oops, unintended: reverted.
